airflow/www/static/js/dag/details/Header.tsx

```diff
@@ -34,6 +34,7 @@ import RunTypeIcon from "src/components/RunTypeIcon";
 import BreadcrumbText from "./BreadcrumbText";
 
 const dagId = getMetaValue("dag_id");
+const dagDisplayName = getMetaValue("dag_display_name");
```

Comment:
We actually have to add this to the meta tags of `dag.html`, right now this is blank.